notebooks/concepts/imviz-wcs-rotation.ipynb

```diff
@@ -0,0 +1,223 @@
+    "    cache=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "401af77b-907a-4dbc-bdb0-b60501f270e9",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "imviz = Imviz()\n",
+    "\n",
+    "paths = [\n",
+    "    [cwd + 'jw02731-o001_t017_nircam_clear-f200w_i2d.fits', 'JWST/NIRCam F200W'],\n",
+    "    [cwd + 'jw02731-o001_t017_nircam_clear-f187n_i2d.fits', 'JWST/NIRCam F187W'],\n",
+    "    [cwd + 'j9dk11010_drz.fits', 'HST/ACS F658N (1)'],\n",
+    "    [cwd + 'j9dk11020_drz.fits', 'HST/ACS F658N (2)'],\n",
+    "    [tess_path, 'TESS FFI']\n",
+    "]\n",
+    "load_order = [0, 1, 2, 3, 4]\n",
+    "\n",
+    "with imviz.batch_load():\n",
+    "    for path, data_label in np.array(paths)[load_order]:\n",
+    "        imviz.load_data(path, data_label=data_label)\n",
+    "\n",
+    "po = imviz.plugins['Plot Options']\n",
+    "po.image_color_mode = 'Monochromatic'\n",
+    "\n",
+    "colors = np.array(['r', '#40eb34', '#03dbfc', '#03dbfc', 'y'])[load_order]\n",
+    "vmins = np.array([1.2, 5, 0, 0, 0])[load_order]\n",
+    "vmaxes = np.array([8, 150, 3, 3, 3e4])[load_order]\n",
+    "alphas = np.array([1, 1, 1, 1, 0.7])[load_order]\n",
+    "\n",
+    "zip_items = zip(\n",
+    "    imviz.app.data_collection, colors, alphas, vmins, vmaxes\n",
+    ")\n",
+    "\n",
+    "for i, (data, color, alpha, vmin, vmax) in enumerate(zip_items):\n",
+    "    if not data.label.startswith(\"CCW\"):\n",
+    "        po.layer = data.label\n",
+    "        po.stretch_function = 'arcsinh'\n",
+    "        po.image_opacity = alpha\n",
+    "        po.image_contrast = 1.15\n",
+    "        po.image_color = color\n",
+    "        po.stretch_vmin = vmin\n",
+    "        po.stretch_vmax = vmax\n",
+    "\n",
+    "imviz.link_data(link_type='wcs')\n",
+    "imviz.show('sidecar:split-right', height=1200)\n",
+    "\n",
+    "near_nebula = SkyCoord(159.22, -58.63, unit=u.deg)\n",
+    "imviz.default_viewer.center_on(near_nebula)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "506f619b-f0a0-495c-8dd3-eec9a5d94896",
+   "metadata": {},
+   "source": [
+    "### Generate a set of WCS-only layers\n",
+    "\n",
+    "Load the presets for WCS-only layers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f3a4f56-305e-4010-9f95-68b369737463",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "lc = imviz.plugins['Links Control']\n",
+    "\n",
+    "# make option for: North-up, East left\n",
+    "lc._obj.create_north_up_east_left()\n",
+    "\n",
+    "# make option for: North-up, East right\n",
+    "lc._obj.create_north_up_east_right()\n",
+    "\n",
+    "lc.open_in_tray()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fc97a58-fadd-49d1-8829-da7478f1674e",
+   "metadata": {},
+   "source": [
+    "### Making WCS-only layers visible for debugging\n",
```

Comment:
I don't think this works anymore, so this section should be removed.

Reply:
I tested today and it works for me – you have to toggle this boolean:
```diff
- make_wcs_only_layers_visible = False
+ make_wcs_only_layers_visible = True
```